test/feature_discovery_test.dart

```diff
@@ -122,7 +122,7 @@ void main() {
 
     // Declares what OverflowMode's should allow the button to be tapped.
     const Map<OverflowMode, bool> modes = <OverflowMode, bool>{
-//      OverflowMode.ignore: false,
+      OverflowMode.ignore: false,
```

Comment:
Fixed what I was talking about in https://github.com/axel-op/feature_discovery/pull/35#issuecomment-536239443.